package.json

```
@@ -20,7 +20,7 @@
     "webpack-cli": "^5.0.0"
   },
   "dependencies": {
-    "@fastly/js-compute": "^0.5.12"
+    "@fastly/js-compute": "^1.1.0"
   },
   "scripts": {
```

This is also required when not using a separate bundler with js-compute in order to support `await`
```suggestion
  },
  "type": "module",
  "scripts": {
```

---

ah, thanks for the comment.  however, this starter kit uses webpack. we'll make a separate webpack-less starter kit for that